packages/components/Menu/src/MenuGroup/MenuGroup.tsx

```diff
@@ -39,7 +44,21 @@ export const MenuGroup = compose<MenuGroupType>({
         }
         return child;
       });
-      return <Slots.root {...mergedProps}>{childrenWithSet}</Slots.root>;
+
+      return (
+        <Slots.root {...mergedProps}>
+          <Slots.focusZone
+            // avoid error that fires when props are passed into React.fragment
+            {...(hasFocusZone && {
+              focusZoneDirection: 'vertical',
+              enableFocusRing: false,
+              isCircularNavigation: true,
```

Comment:
@samuelfreiberg check me if I'm out of date but I believe `isCircularNavigation` can be replaced with `NavigateAtEnd='NavigateWrap'`

Reply:
We expose the prop as isCircularNavigation, so this is correct. But under the hood, you are correct.